src/Abstractions/NexusMods.Abstractions.NexusModsLibrary/Models/CollectionRevision.cs

```diff
@@ -0,0 +1,54 @@
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.Abstractions.NexusModsLibrary.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Attributes;
+using NexusMods.MnemonicDB.Abstractions.Models;
+
+namespace NexusMods.Abstractions.NexusModsLibrary.Models;
+
+/// <summary>
+/// Metadata about a collection on Nexus Mods.
+/// </summary>
+public partial class CollectionRevision : IModelDefinition
```

Comment:
For consistency, this should be named like the other models in the project. Also the doc comment on this is the exact same as on `Collection`.

Reply:
See the comment above about collection naming